core/src/stores/migrations/202412011_indices_parents.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX CONCURRENTLY idx_parents_second ON data_sources_nodes (data_source, (parents[2]));
+CREATE INDEX CONCURRENTLY idx_parents_single ON data_sources_nodes (data_source, (array_length(parents, 1) = 1));
```

Comment:
include the name of the table in the index : 
```suggestion
CREATE INDEX CONCURRENTLY idx_data_sources_nodes_parents_second ON data_sources_nodes (data_source, (parents[2]));
CREATE INDEX CONCURRENTLY idx_data_sources_nodes_parents_single ON data_sources_nodes (data_source, (array_length(parents, 1) = 1));
```